src/components/common/S3Upload/RenderS3Upload.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { postNewUpload, postSubmissionPage } from '../../../api';
+import { UploadOutlined } from '@ant-design/icons';
+import { Button, Upload, message } from 'antd';
+import { useState } from 'react';
+
+export default function S3UploadButton() {
+  const [file, setFile] = useState([]);
+  const [uploading, setUploading] = useState(false);
+
+  const fileSelected = fileName => {
+    const selectedFile = fileName.file;
+    setFile(selectedFile);
+    console.log('file', selectedFile);
```

Comment:
let'slets remove the console.log entries

Reply:
removed console.log entries.